drivers/CMakeLists.txt

```diff
@@ -33,6 +34,7 @@ SOURCES
     src/scd40.cpp
     src/soil_sensor_sht21.cpp
     src/sk9822.cpp
+    src/drv8825.cpp
 
     PACKAGES
 #    libhal-__device__
```

Comment:
```suggestion
```

Delete this line.

Reply:
The comment has been updated to clarify the template. look over it to see if it is clear.